server/base/src/test/java/org/apache/accumulo/server/security/delegation/AuthenticationTokenSecretManagerTest.java

```diff
@@ -208,6 +208,8 @@ public void testVerifyPassword() throws Exception {
     assertArrayEquals(password, secretManager.retrievePassword(id));
 
     // Make a second token for the same user
+    // Sleep for 1 millisecond to guarantee token is unique
+    Thread.sleep(100);
```

Comment:
The comment doesn't match. This is sleeping for 100 milliseconds, right? Other than that mismatch, this seems fine.

```suggestion
    // Briefly sleep to guarantee token is unique, since the token is based on the time
    Thread.sleep(100);
```

Reply:
Yeah I updated the time from 1 to 100 in my second commit but didn't update the comment so good catch.